README.md

```diff
@@ -1,95 +1,28 @@
-
-    {
-        "description": "Test subscription",
-        "subject": {
-            "entities": [
-            {
-                "idPattern": ".*",
-                "type": "Room"
-            }
-            ],
-            "condition": {
-                "attrs": [
-                "temperature",
-                ]
-            }
-        },
-        "notification": {
-            "http": {
-                "url": http://quantumleap:8668/notify
-            },
-            "attrs": [
-            "temperature",
-            ],
-            "metadata": ["dateCreated", "dateModified"]
-        },
-        "throttling": 5
-    }
-
-Important thing to notice here is the presence of the ```"metadata": ["dateCreated", "dateModified"]``` part. This tells Orion to include the modification time of the attributes, and this time will be used as the time index in the database. If this is somehow missing, Quantumleap will use its current system time at which the notification arrives instead.
-
-The notification is fed to a translator, which by default is for CrateDB as mentioned before.
-
-In addition to the */notify* endpoint, the API will include endpoints for advanced raw and aggregated data retrieval.
-
-The overall picture of the main pieces is shown below.
-
-![Alt text](https://g.gravizo.com/svg?@startuml;skinparam%20componentStyle%20uml2;!define%20ICONURL%20https://raw.githubusercontent.com/smartsdk/architecture-diagrams/smartsdk-template/dist;!includeurl%20ICONURL/common.puml;!includeurl%20ICONURL/fiware.puml;!includeurl%20ICONURL/smartsdk.puml;interface%20NGSI;FIWARE%28cb,%22Context%20Broker%20\n%20-%20Orion%22,component%29;[Sensor%20@%20IoT%20Layer]%20-right-%20NGSI;NGSI%20-right-%20cb;package%20%22Quantumleap%22%20{;SMARTSDK%28api,%22API%22,component%29;SMARTSDK%28reporter,%22Reporter%22,component%29;SMARTSDK%28translator,%22Translator%22,component%29;api%20-up-%20NGSI;api%20%3C-down-%3E%20translator;api%20%22/notify%22%20-down-%3E%20reporter;reporter%20-right-%3E%20translator;};[CrateDB]%20%3C-left-%20translator;[Grafana]%20-down-%20CrateDB;@enduml;)
-
-## Important things to keep in mind
-
-- Quantumleap needs to know where CrateDB is deployed. You can set this with the variable ```CRATE_HOST```. e.g ```CRATE_HOST=http://0.0.0.0:4200``` 
-
-- Notifications must come in complete [NGSI JSON Entity Representation](http://docs.orioncontextbroker.apiary.io/#introduction/specification/json-attribute-representation) form. Other forms, such as simplified entity representation, are not supported by QL because they lack information on attribute types, required by QL to make proper translations.
+Quantumleap is an adapter aimed to bring [FIWARE NGSIv2](http://docs.orioncontextbroker.apiary.io/#) Historical Data on top of TimeSeries Databases.
 
+Want to know more? Head to the docs by clicking [here](https://smartsdk.github.io/ngsi-timeseries-api/).
```

Comment:
Want to know more? Head to the [docs](https://smartsdk.github.io/ngsi-timeseries-api/).